src/Setup/WixBA/RootView.xaml

```diff
@@ -1,125 +1,487 @@
-                    </EventTrigger>
-                </Image.Triggers>
-            </Image>
-
-            <!-- Up To Date -->
-            <Button Grid.Row="2" Grid.Column="0" DataContext="{Binding UpdateViewModel}" IsEnabled="False" Visibility="{Binding IsUpToDate, Converter={StaticResource BooleanToVisibilityConverter}}">Up To Date</Button>
-            <Image Grid.Row="2" Grid.Column="0" DataContext="{Binding UpdateViewModel}" Visibility="{Binding IsUpToDate, Converter={StaticResource BooleanToVisibilityConverter}}" Source="resources\wrench.png"/>
-
-            <!-- Update Available -->
-            <Button Grid.Row="2" Grid.Column="0" DataContext="{Binding UpdateViewModel}" Command="{Binding UpdateCommand}" Visibility="{Binding UpdateEnabled, Converter={StaticResource BooleanToVisibilityConverter}}">Update Available</Button>
-            <Image Grid.Row="2" Grid.Column="0" Source="resources\wrench.png" DataContext="{Binding UpdateViewModel}" Visibility="{Binding UpdateEnabled, Converter={StaticResource BooleanToVisibilityConverter}}"/>
-
-            <!-- News -->
-            <Button Grid.Row="2" Grid.Column="1" DataContext="{Binding InstallationViewModel}" Command="{Binding LaunchNewsCommand}" Background="#FFCB0000">News</Button>
-            <Image Grid.Row="3" Grid.Column="1" Source="resources\news.png"/>
-
-            <!-- Exit -->
-            <Button Grid.Row="2" Grid.Column="2" Background="#FFCB0000" DataContext="{Binding InstallationViewModel}" Visibility="{Binding ExitEnabled, Converter={StaticResource BooleanToVisibilityConverter}}" Command="{Binding CloseCommand}">Exit</Button>
-            <Image Grid.Row="2" Grid.Column="2" Source="resources\exit.png" DataContext="{Binding InstallationViewModel}" Visibility="{Binding ExitEnabled, Converter={StaticResource BooleanToVisibilityConverter}}" />
-
-            <!-- Cancel -->
-            <Button Grid.Row="2" Grid.Column="2" Background="#FFCB0000" DataContext="{Binding}" Visibility="{Binding CancelEnabled, Converter={StaticResource BooleanToVisibilityConverter}}" Command="{Binding CancelCommand}">Cancel</Button>
-            <TextBlock Grid.Row="2" Grid.Column="2" HorizontalAlignment="Center" VerticalAlignment="Center" FontSize="60" Foreground="White" IsHitTestVisible="False" DataContext="{Binding}"  Visibility="{Binding CancelEnabled, Converter={StaticResource BooleanToVisibilityConverter}}">X</TextBlock>
-
-        </Grid>
-
-    </Grid>
+    <i:Interaction.Behaviors>
+        <ux:WinNoSnapBehavior />
+    </i:Interaction.Behaviors>
+
+    <Border BorderBrush="{DynamicResource {x:Static SystemColors.ActiveBorderBrushKey}}" BorderThickness="0">
+        <!--  Catch MouseLeftButtonDown for Window dragging.  -->
+        <Border HorizontalAlignment="Stretch"
+                VerticalAlignment="Stretch"
+                Background="{DynamicResource {x:Static SystemColors.WindowBrushKey}}"
+                MouseLeftButtonDown="Background_MouseLeftButtonDown">
```

Comment:
The old UI allowed dragging from clicking on the background since it didn't have the traditional window chrome.  We shouldn't be supporting this anymore.